ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -186,7 +186,7 @@ export const Homepage = withUserProfile()(class extends React.Component<Props, S
         reload();
       }, 10000);
     } else {
-      if (!profile.betaAccessRequestTime) {
+      if (serverConfigStore.getValue().enableBetaAccess && !profile.betaAccessRequestTime) {
```

Comment:
Good catch

Reply:
This was Joel's catch actually – this PR builds off of his https://github.com/all-of-us/workbench/commit/66a85ec70ecdc7984ed82f89b1aff8966de0d9c2